src/methods/method_overloading/hard/q4/RectangleOverload.java

```diff
@@ -0,0 +1,53 @@
+package methods.method_overloading.hard.q4;
+
+/*
+Write a Java Program that prompts the user to enter two integers and two doubles. Then write four overloaded methods
+called rectangleArea. The first takes both integer values for each side of the rectangle. The second takes the first
+integer and the first double for each side of the rectangle. The third takes the second integer and uses it for all
+sides of the rectangle. The fourth takes the second double and uses it for all sides of the rectangle.
+
+Sample Output:
+  Enter two integers:
+  2 4
+  Enter two doubles:
+  5.3 3.3
+  The area of the rectangle is: 8
```

Comment:
We should also maybe change these to be like area of the first rectangle, second, etc. so the students have a better idea of what order they should print them in.